README.md

```diff
@@ -26,13 +26,11 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install px-timeline-service-client
+    $ gem install px-kinesis-client
 
 ## Usage
 
-Follow the documentation wiki here:
 
-* [Development Setup HOWTO](https://github.com/500px/500px/wiki/HOWTO:-Timeline-Service)
```

Comment:
This is a link to our private repo and isn't appropriate here.  You should include code usage of this (ie, subclass this, configure it, queue records).